components/cli/vendor/github.com/docker/docker/pkg/gitutils/gitutils.go

```diff
@@ -15,18 +15,24 @@ import (
 	"github.com/pkg/errors"
 )
 
+type gitRepo struct {
```

Comment:
cause of this discrepancy is same reason as https://github.com/docker/docker-ce/pull/116#discussion_r130778764: PR #100 should have had another commit to update `components/cli/vendor`